### ISSUE TITLE
[SÃO PAULO] Data Science na [Creditas]

### ISSUE THREAD

**Issue by PedrodaIntera** [datascience-br/vagas#3]
## Descrição da vaga
Você domina Python, machine learning, data visualization e sabe que dados só tem seu devido valor se providos de interpretação, diálogo e colaboração com o time?
Ser do time de Data Science da Creditas é mais do que lidar com grandes volumes de dados, tendo o desafio de criar modelos eficientes e capazes de gerar insights acionáveis: é ser responsável por integrar times de analytics, marketing, produto e engenharia em prol da resolução de problemas.

Responsabilidades e atividades:
Utilizar modelos de machine learning e estatística para construir soluções para o negócio
Analisar e identificar tendências e comportamentos com o uso de modelos e estatística
Apresentar os resultados para o time e a equipe de stakeholders
Acompanhar o desempenho dos modelos de Machine Learning em produção, cuidando para que eles estejam sempre cumprindo com os objetivos da empresa
Participar da integração entre times de tecnologia, produtos, analytics e operação para entender e solucionar os projetos da área de Data Science

## Local
Av. Engenheiro Luís Carlos Berrini, 105, 12º andar – Itaim Bibi, CEP 4571010, São Paulo – SP

## Benefícios
- Vale transporte
- Vale refeição
- Assistência médica
- Assistência odontológica
- Seguro de vida
- Estacionamento 
...
## Diferenciais
- Licença-maternidade é de 6 meses e a licença-paternidade de 20 dias, . O benefício também vale para papais e mamães que adotarem ou obtiverem a guarda judicial da criança e para casais homoafetivos;
- Gympass
-  Auxílio educação/estudo e Aulas de Inglês in company
- Ambiente que valoriza feedback e 1:1;
- Ambiente dinâmico e com crescimento exponencial

#### Obrigatórios:
- Python
- Experiência com bibliotecas como sklearn, pandas, scipy, numpy e xgboost
- Experiência em criar gráficos utilizando seaborn e matplotlib
- Conhecimento de banco de dados (PostgreSQL, MySQL)
- Ter a capacidade de traduzir a linguagem de um cientista de dados para a de outros profissionais

#### Diferenciais:
- Ter trabalhado em projetos com grandes volumes de dados, SHAP Values e aprendizado por reforço

## Contratação
- Efetivo CLT (presencial em tempo integral) – 08h ás 17h/ 09h ás 18h/ 10h ás 19h

## Nossa empresa
A Creditas é a principal plataforma online de crédito com garantia do Brasil e trabalha com dois produtos principais: o empréstimo com garantia de imóvel e o empréstimo com garantia de veículo. Essa modalidade de crédito se diferencia porque oferece taxas mais baixas, parcelas menores, maiores valores concedidos e prazos de pagamentos mais longos. Quando o cliente coloca seu bem como garantia, a operação se torna mais segura e a Creditas pode oferecer juros mais baixos. O DNA digital da Creditas permite usar a tecnologia para escalar o negócio, otimizar processos e melhorar a experiência dos clientes – sem dispensar o atendimento personalizado, prestando consultoria e oferecendo educação financeira

## Como se candidatar
Inscrições através no link: http://bit.ly/data-scientist-creditas
Dúvidas através do email: pedro@byintera.com

OBS: não esqueça de colocar os labels
#alocado
#CLT
#SP
